src/common/components/clusterConfiguration/OpenShiftVersionSelect.tsx

```diff
@@ -37,15 +38,17 @@ const OpenShiftVersionSelect: React.FC<OpenShiftVersionSelectProps> = ({ version
     values: { highAvailabilityMode, openshiftVersion },
     setFieldValue,
   } = useFormikContext<ClusterCreateParams>();
-
+  const supportLevelService = React.useContext(FeatureSupportLevelContext);
   const selectedVersion = versions.find((v) => v.value === openshiftVersion) || versions[0];
   React.useEffect(() => {
     if (highAvailabilityMode === 'None' && !isSNOSupportedVersion(selectedVersion)) {
       const firstSupportedVersionValue = versions.find(isSNOSupportedVersion)?.value;
       setFieldValue('openshiftVersion', firstSupportedVersionValue);
     }
   }, [highAvailabilityMode, selectedVersion, setFieldValue, versions]);
-
+  React.useEffect(() => {
+    supportLevelService.setOpenshiftVersion(openshiftVersion);
+  }, [supportLevelService, openshiftVersion]);
```

Comment:
The `supportLevelService` should refer be always the same instance after the context is initialized. AFAIU we are not creating a different instance during the app's lifecycle, it is a singleton and it remains such until the app ends, correct?

```suggestion
  }, [openshiftVersion]);
```